server/web/errors/index.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+exports.register = function(server, options, next) {
+
+  server.ext("onPreResponse", function (request, reply) {
+
+    var response = request.response;
+    if (!response.isBoom) {
+        return reply.continue();
+    }
+
+    // Replace error with friendly HTML
+    var error = response;
```

Comment:
I am using the var names as a hint of what has been inferred as the response is being checked. 

So in other words, once we know ``` !request.response.isBoom ``` that fails, then we are dealing with an error, but before that we were dealing with a generic response.